docs/INSTALL.MD

```diff
@@ -1,18 +1,90 @@
-## Installation
+# Installation
 
 JuliaBox is developed and tested on Ubuntu 14.04. But it should be easy to adapt to any Linux environment supporting Docker.
 
-### Single Machine Development Setup
-1. Get JuliaBox sources:
-    - If `git` is not installed yet, run: `sudo apt-get install git`
-    - `git clone https://github.com/tanmaykm/JuliaDockerImages.git`
-    - `git clone https://github.com/JuliaLang/JuliaBox.git`
-2. Install docker and a few more packages required to develop and administer JuliaBox:
-    - Modify `JuliaBox/scripts/install/sys_install.sh` and set `DOCKER_FS` to `AUFS` instead of `DEVICEMAPPER`.
-        - This will avoid certain unnecessary configurations required only in certain production environments.
-    - Run `JuliaBox/scripts/install/sys_install.sh`
-3. Create work folders: `mkdir -p /jboxengine/conf /jboxengine/data /jboxengine/data/db`
-4. Configure JuliaBox:
+## Single-machine setup
+
+### Installation overview
+Installing JuliaBox on a single machine consists of five principle steps, the specifics of which vary depending upon the desired configuration. The following is a general overview of the process, and the key scripts and files involved. It may be useful to consult the included `Architecture.MD` document to gain a high-level overview of JuliaBox before reading these instructions.
```

Comment:
may be better to link references directly. i.e. [architecture document](Architecture.md)